service/src/test/java/com/ge/predix/controller/test/PolicyEvalWithGraphDbControllerIT.java

```diff
@@ -178,7 +211,7 @@ public void testPolicyEvaluation(final Zone zone, final PolicySet testPolicySet,
     Object[] scopedAttributeInheritanceData() {
         return new Object[] { this.testZone1, this.policySet, createTwoParentResourceHierarchy(),
                 createScopedSubjectHierarchy(), createPolicyEvalRequest("GET", EVIDENCE_IMPLANT_ID, AGENT_MULDER),
-                Effect.DENY };
+                Effect.DENY, null };
```

Comment:
We don't need to add an additional parameter that affects all the tests in the class. We can create a separate data-provider object for the traversal-limit tests if needed.